react/features/blur/functions.js

```diff
@@ -16,3 +16,21 @@ export function getBlurEffect() {
 
     return loadScript('libs/video-blur-effect.min.js').then(() => ns.effects.createBlurEffect());
 }
+
+/**
+ * Checks context filter support.
+ *
+ * @returns {boolean} True if the filter is supported and false if the filter is not supported by the browser.
+ */
+export function checkFilterSupport() {
+    const canvas = document.createElement('canvas');
+    const ctx = canvas.getContext('2d');
+
+    if (ctx.filter === 'undefined') {
```

Comment:
This doesn't work, it's comparing the string value. You need to either check with `typeof` or test directly `undefined` (no quotes).

Reply:
Also, let's stop the value of the test in a variable outside the function so we don't need to recompute it with every call to this function.